charset.c

```diff
@@ -55,7 +55,9 @@ static const struct
   const char *key;
   const char *pref;
 }
-PreferredMIMENames[] = 
+
+// clang-format off
```

Comment:
I don't think it's a big thing to keep this code aligned. First thing, I actually never saw that code properly aligned, because I never had a tabstop at 8 (I default to 4), as you can see below it's already crap. 

Second thing, it's that the MIME names list isn't something that's likely to change anytime soon. Those line of codes have been committed by Thomas Roessler 17 years ago, and has never been changed since.